grazer/src/main/java/edu/asu/diging/grazer/core/model/impl/Concept.java

```diff
@@ -303,4 +303,11 @@ public OffsetDateTime getLastUpdated() {
     public void setLastUpdated(OffsetDateTime lastUpdated) {
         this.lastUpdated = lastUpdated;
     }
+
+    @Override
+    public int compareTo(IConcept o) {
+        // TODO Auto-generated method stub
+        return 0;
+    }
```

Comment:
compareTo method should contain the logic that be used to compare the elements of type Concept. The return value 0 means "equal": All Concepts will be treated as equal. The comparison should be based on a property such as uri or id. Usually, the properties used in the overriden equals() method is used in the compareTo() method

Reply:
The HomeController is sorting using the comparator implemented in that class, so I have deleted this since it is never being used.